.travis.yml

```diff
@@ -17,19 +14,36 @@ cache:
     - $HOME/.composer/cache/files
 
 env:
-  - SYMFONY_VERSION='2.8.*'
-  - SYMFONY_VERSION='3.0.*'
-  - SYMFONY_VERSION='3.1.*'
+  - SYMFONY_VERSION='3.3.*'
+
+matrix:
+  include:
+    - php: 5.5
+      env:
+        - COMPOSER_FLAGS="--prefer-lowest"
+        - SYMFONY_VERSION='2.8.*'
+    - php: 7.1
+      env:
+      env:
+        - SYMFONY_VERSION='2.8.*'
+    - php: 7.1
+      env:
+        - SYMFONY_VERSION='3.1.*'
+    - php: 7.1
+      env:
+        - SYMFONY_VERSION='3.2.*'
+    - php: hhvm
+      dist: trusty
 
 before_install:
   - phpenv config-rm xdebug.ini || true
-  - php -i | grep "ICU version"
```

Comment:
this fails with hhvm. looks like a debug leftover. if its important, i can wrap it into some check if we are on hhvm or not.